batch/testResultsCollector.js

```diff
@@ -2,39 +2,73 @@ import {XUnitParser} from './parsers/xUnitParser';
 import mongoose from 'mongoose';
 import {TestSuiteModel} from '../models/testSuiteModel';
 import {TestCaseModel} from '../models/testCaseModel';
-
+import {GithubHelper} from '../helpers/githubHelper';
 
 class TestResultsCollector {
 	constructor(configuration) {
 		this.configuration = configuration;
```

Comment:
Please document this configuration, either through doc, or through default parameters, or through parameter naming and validation.